packages/fe-fpm-writer/test/unit/column.test.ts

```diff
@@ -70,7 +70,7 @@ describe('CustomAction', () => {
             expect(fs.read(expectedFragmentPath)).toMatchSnapshot();
         });
 
-        test('version 1.86, with new handler, all properties', () => {
+        test('version 1.86.3, with new handler, all properties', () => {
```

Comment:
Why does this test case need to be changed?

Reply:
Just to show that is also works with a different patch level.